### PR TITLE
Upgrading ldapjs version

### PR DESCRIPTION
**Pull request by ihavenoidea14**
I'm new at this, so if I need to make any changes I'd be happy to.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,9 +33,13 @@
     "bluebird": "^3.5.0",
     "bluebird-co": "^2.1.2",
     "joi": "^10.5.2",
-    "ldapjs": "^1.0.0"
+    "ldapjs": "^1.0.1"
   },
   "devDependencies": {
+    "eslint-plugin-import": "^2.7.0",
```
Comment (jsumners) [inline package.json]: These dependencies are unnecessary. Please remove them.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adldap",
-  "version": "3.1.2",
+  "version": "3.1.3",
```
Comment (jsumners) [inline package.json]: Please do not update the version. That is for release maintainers (i.e. me) to do.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,5312 @@
+{
```
Comment (jsumners) [inline package-lock.json]: Please remove this `package-lock.json` file.

**Review by jsumners — CHANGES_REQUESTED** [review]

**Comment by ihavenoidea14** [comment]
Ok, sorry for the sloppiness. Great module by the way, very useful.

**Review by jsumners — APPROVED** [review]

**Comment by jsumners** [comment]
No problem. Corrections are easy to make. Thank you for the PR.

**Pull request merged**